packages/framework/src/Model/Feed/FeedExportFactory.php

```diff
@@ -67,10 +67,10 @@ public function __construct(
     /**
      * @param \Shopsys\FrameworkBundle\Model\Feed\FeedInterface $feed
      * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domainConfig
-     * @param string|null $lastSeekId
+     * @param int|null $lastSeekId
      * @return \Shopsys\FrameworkBundle\Model\Feed\FeedExport
      */
-    public function create(FeedInterface $feed, DomainConfig $domainConfig, ?string $lastSeekId = null): FeedExport
+    public function create(FeedInterface $feed, DomainConfig $domainConfig, ?int $lastSeekId = null): FeedExport
```

Comment:
This is BC break

Reply:
But actually it is a fix of type mismatch. See the only usage we have and the variable previous type hint. https://github.com/shopsys/shopsys/blob/762a90cc3543b33bce1daca2feb8c48adcd3e79a/packages/framework/src/Model/Feed/FeedFacade.php#L87

Reply:
So remove the typehint and use deprecation annotation to describe what should be done in next major